samples/samples/pg_snippets.py

```diff
@@ -0,0 +1,1542 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to do basic operations using Cloud
+Spanner PostgreSql dialect.
+
+For more information, see the README.rst under /spanner.
+"""
+import argparse
+import base64
+import datetime
+import decimal
+import time
+
+from google.cloud import spanner, spanner_admin_database_v1
+from google.cloud.spanner_admin_database_v1.types.common import DatabaseDialect
+from google.cloud.spanner_v1 import param_types
+
+OPERATION_TIMEOUT_SECONDS = 240
+
+
+# [START spanner_postgresql_create_instance]
```

Comment:
A pg instance is not required separately. You can reuse the same instance as google sql instance, and delete this test